packages/app-admin-ui/client/components/CreateItemModal.js

```diff
@@ -129,16 +131,41 @@ const CreateItemModal = ({ prefillData = {}, onClose, onCreate, viewOnSave }) =>
     }
   });
 
-  const _onClose = () => {
-    if (loading) return;
+  // Identifies if the user has changed the initial data in the form.
+  const hasFormDataChanged = () => {
```

Comment:
Can you make this use `useCallback`

Reply:
This may not require useCallback decorator as it's not being passed as a prop.